### DISCUSSION TITLE
Meeting note #7, 2022-11-29

### DISCUSSION THREAD

**Discussion by jongfeel** [wisdom08/dev-community#42] [General]
### 최근 작업 내용 리뷰에 대한 추가 의견

- 기능 구현이 전부인 코딩 말고 사용자가 써보는 기능으로 범위를 넓혀야 함
- QA가 왜 중요한지 모르고, 이슈를 잔뜩 개발자의 적 쯤으로 생각하면 큰 잘못임
- 백엔드 중심의 정적인 페이지이므로 삭제한 페이지 번호를 브라우저 히스토리가 알고 있고 페이지를 다시 뒤로 보내면 whitelabel error가 발생
- 역시 게시글 수정에서 수정 내용을 더 적고 비밀번호를 다르게 적으면 수정해서 적었던 내용이 없어짐 - 사용성 문제

### 회사 업무 파악

루비나 코틀린 이런 쪽을 파볼게 아니라 기획,설계,디자인 문서를 보고 어떤 소프트웨어를 개발하는지 파악
그 이후에 그 기능을 구현하기 위해 어떤 코드를 짰는지 파악하는게 좋은 방법

고객이 누구인지 한번 확인해 보면, 개발을 하는데 의미가 있는 시간을 보내는지 아닌지 확인해 볼 수 있음
영업 채널에서 고객 무시하고 해달라는 요청 안듣고 하는 회사에서 개발 해봤자
의미있는 소프트웨어를 만들고 있다고 보기 어려움

### 회사 사람들 파악

모두가 코딩은 잘 할거고
주어진 일도 열심히 할거고
개발 얘기도 다들 잘 할 것.

그 외에 잘 하기 위해 뭘 하고 있는지 살펴보기.
회사에서 보내는 시간 외에 더 개발을 잘하기 위해 보내는 시간 체크
그런걸 다른 사람들에게 보여주고 얘기하고 피드백 주고 받고 하는 걸 하는 사람이 있다?
그런 사람하고 친해져야 함.

차선책으로 회사 일도 뭔가 다르게 생각하고 적용해서 
책에서 나온 이론적인 것들을 실제 자신의 업무에 해보고 공유하는 사람이 있다?
꽤 괜찮은 사람일 겁니다.

그 외에 다들 착하고 일 열심히 하고 좋은 사람들이라는 수식어는 얼마든지 붙일 수 있습니다.

**Comment by wisdom08** [comment]
감사합니다. 멘토님!
꽤 괜찮은 사람과 친해지고 싶기도 하지만 저도 그런 괜찮은 사람이 될 수 있도록 관리를 잘 해보겠습니다...ㅎㅎㅎ
